src/Orleans.Core/Configuration/Options/AdoNetOptions.cs

```diff
@@ -16,4 +22,25 @@ public class AdoNetOptions
         /// </summary>
         public string InvariantForReminders { get; set; }
     }
+
+    public class AdoNetOptionsFormatter : IOptionFormatter<AdoNetOptions>
+    {
+        public string Category { get; }
```

Comment:
Is this supposed to be set somewhere?

Reply:
I don't recall what category is for.  I'll check with Xiao.

Reply:
Category is for classifying named options. Not necessary if not a named option.